mcs/class/System/Test/System.Diagnostics/PerformanceCounterTest.cs

```diff
@@ -0,0 +1,210 @@
+// TODO reconcile PerformanceCounterTest.c
+// and PerformanceCounterTests.cs
+
+// from MSDN
+// https://msdn.microsoft.com/en-us/library/system.diagnostics.countercreationdata(v=vs.110).aspx
+
+#if !NO_NUNIT
+using NUnit.Framework;
+#endif
+using System;
+using System.Threading;
+using System.Diagnostics;
+using System.Collections;
+using System.Collections.Specialized;
+
+namespace MonoTests.System.Diagnostics
+{
+#if !NO_NUNIT
+	[TestFixture]
+#endif
+	public class PerformanceCounterTest
+	{
+
+#if NO_NUNIT
+	public static void Main()
+	{
+		if (verbose)
+			Console.WriteLine("1");
+		new PerformanceCounterTest().A();
+	}
+#endif
+
+	public bool verbose = Environment.GetEnvironmentVariable("V") != null;
+
+#if !NO_NUNIT
+	[Test]
+#endif
+	public void A()
```

Comment:
Use sensible names for each tests, instead of single method with combined checks

Reply:
+1

Reply:
This was all copied from elsewhere and minimally changed.